### DISCUSSION TITLE
List organizations for the authenticated user returns empty response

### DISCUSSION THREAD

**Discussion by shan2new-clearfeed** [community/community#45996] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

**Request**:  
```
curl \
    -H "Accept: application/vnd.github+json" \
    -H "Authorization: Bearer <your_token> "\
    -H "X-GitHub-Api-Version: 2022-11-28" \
    https://api.github.com/user/orgs
```

**Response**: []

**Expected Response**: List of 2 organisations.

**Steps to reproduce:**
1. Go through OAUTH flow to extract token
2. Use the token and fire the API request.

**Note:**
1. I have already tried with multiple other tokens as well, it doesn't work.

**Comment by jge162** [comment]
Hi @shan2new-clearfeed Make sure that the token has the `read:org` scope, which allows you to read organization information for the authenticated user. 

check the scopes your token has by sending a 
GET request to -> `/applications/{client_id}/token` Replace` {client_id} ` with the ID of the OAuth application that you used to generate the token.

**Reply by shan2new-clearfeed** [reply]
I have already added `read:org` as you have mentioned, it still doesn't work.
Also, the endpoint that you have provided gives 404 error. `https://api.github.com/applications/<client_id>/token`

**Reply by jge162** [reply]
check the scopes that have been granted to your token by examining the response of the -> 

`/app/installations/<installation_id>/access_tokens` endpoint.

 If the necessary scopes are not present, you may need to go through the `OAuth flow` again and make sure that the desired -> `scopes are included.`

**Reply by shan2new-clearfeed** [reply]
Got the following response from the endpoint. Also by installation, do you mean 
```
{
    "message": "A JSON web token could not be decoded",
    "documentation_url": "https://docs.github.com/rest"
}
```

**Reply by jge162** [reply]
installation refers to a specific instance of a GitHub App installed on a repository or an organization. When a GitHub App is installed, it has access to resources such as repositories, issues, pull requests, and more

Each installation has its own unique identifier, which is called the installation ID.


> Got the following response from the endpoint

The error message you're encountering indicates that the JSON Web Token (JWT) you're using in the Authorization header is not correctly formatted or is invalid

**Reply by logan-at-placer** [reply]
I was encountering the same issue as OP, but believe it is because the app was not approved by the org, which I believe is a requirement before the read:org scope can see the org(s), as shown here:

![image](https://user-images.githubusercontent.com/119893332/235239225-06b3c3c8-f125-4d43-8685-84ad658a4f78.png)

@shan2new-clearfeed - were you able to determine a solution for the issue you were facing?

Edit: I can confirm that after getting the org to approve the application, the data returned was no longer empty 🎉
